app/views/users/new.html.erb

```diff
@@ -1,7 +1,6 @@
 <h1><%= t(".title") %></h1>
 <%= simple_form_for @user do |f| %>
-  <%= f.input :name %>
-  <%= f.input :email %>
+  <%= render "form", f: f %>
```

Comment:
If I understand correctly, you have pulled out the two form fields out because they are the only thing that is the same in both forms.
But in this case I would prefer the duplication, rather than the confusion of what is happening here.

Also, if you go through and abstract an admin name space, you will then be able to extract the whole form into a partial.

Reply:
Will revisit the form partial after I implement admin scope. Done sha: 78a6e82